firmware/application/apps/ui_text_editor.cpp

```diff
@@ -115,8 +115,10 @@ bool TextViewer::on_encoder(EncoderEvent delta) {
 
     if (cursor_.dir == ScrollDirection::Horizontal)
         updated = apply_scrolling_constraints(0, delta);
-    else
+    else {
+        delta *= 16;
```

Comment:
Not really related to the rest of this PR -- I could remove if you want, but it does make it much easier to scroll large files.
The direction keys work fine for within one screen (Thanks to @NotherNgineer's repeat fix) -- this turns the encoder into more of a "page up/down".

Reply:
It's okay, I know you're going to update the wiki page too.
;-p